### DISCUSSION TITLE
[bug] wrong password via graphql api returns type error from swell backend

### DISCUSSION THREAD

**Discussion by jezzzm** [swellstores/community#265] [Help] [answered]
using mutation via `graphql-request` or postman:
```graphql
 mutation Login($email: String!, $passwordToken: String, $password: String) {
    loginAccount(email: $email, passwordToken: $passwordToken, password: $password) {
      success
    }
  }
```

correct password is fine, login works normally. however, providing an incorrect password doesnt return an "invalid credentials" error message, but instead passes on some kind of type error:
```
{
  "errors": [
    { 
    "message":"Cannot destructure property 'id' of '(intermediate value)' as it is null.",
    "locations":[ {"line":2,"column":5} ]
    }
  ],
  "data": {
    "loginAccount":null
  }
}
```

**Comment by sags95** [comment, marked as answer]
Thanks for pointing this out @jezzzm, I'll be able to file this so the team can take a look.

**Reply by jezzzm** [reply]
thank you! it's a bit of an issue atm as we have no way to determine an actual backend error versus incorrect credentials, and therefore present appropriate feedback to our users
